### DISCUSSION TITLE
Probability factor per step

### DISCUSSION THREAD

**Discussion by juanjo1138** [bangcorrupt/hacktribe#87] [Ideas]
In the step edit mode, add probability factor per step.

**Comment by bangcorrupt** [comment]
There's a spare byte in the groove template I'm not sure what to do with...

**Reply by juanjo1138** [reply]
There you have it!

**Reply by Piergap** [reply]
That would be great: you may take inspiration from the similar features on the famous Pajen's hack for Volca Sample as example. That is a great implementation

**Reply by gnutron** [reply]
The pajen os completely w probability makes the sample to easily be the best volca. If the tribe had it, id be so stoked!

**Reply by qioxoip** [reply]
Would be great if it could also work like the elektron and play every 2nd of 2 , 1st of 2, 1st of 4 steps and so on. This would also make up very much for the lack of more than 4 bars.

**Reply by Piergap** [reply]
HI, I could finally install the hacktribe and the increased waveforms and filters are great! I hoped that something was actually done on the sequencer part (....here we were talking about probability per step...) but as far as i can see, that section is unchanged.
Just wanted to be sure if it was finally discharged or if there's any plan on it.
thanks in any case!

**Reply by bangcorrupt** [reply]
Hi, I haven't done anything with sequencing yet, unless you count groove templates.  I'll add groove editing via NRPN before attempting probability.  Nothing will change for a while.

**Comment by chaocrator** [comment]
+1. 
i love probability in Circuit Tracks, and in MC-707/101 (but MC's have it only for drum tracks). 
using it a lot to add variations to patterns. 
would love to have it on E2 as well.
